core/inject.js

```diff
@@ -35,8 +35,10 @@ goog.require('goog.dom');
  * Initialize the SVG document with various handlers.
  * @param {!Element} container Containing element.
  * @param {Object} opt_options Optional dictionary of options.
+ * @param {boolean=} opt_external_css Optional whether a custom css is used and
+ *   Blockly should not inject any css style.
  */
-Blockly.inject = function(container, opt_options) {
+Blockly.inject = function(container, opt_options, opt_external_css) {
```

Comment:
Right before your 'opt_external_css' flag is 'opt_options', a dictionary of configuration flags.  Makes more sense to put your CSS flag in that dictionary.  I've created a patch that does this and submitted it here: https://github.com/google/blockly/commit/6e45f933e73243be8aa9d9884806efd8cd757a35

I'll update the documentation here:
https://developers.google.com/blockly/installation/overview#configuration
With this line:
css: boolean If false, don't inject CSS (providing CSS becomes the document's responsibility). Defaults to true.